### ISSUE TITLE
Options menu hidden label

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1758]
Since the toggle label for the Options component remains consistent (i.e. doesn't change on selection in the menu), we should remove the hidden span that provides the additional label, since this is just redundant.

Instead, we should only include `aria-label` on the examples that include an icon as the toggle button. And the menu should have `aria-labelledby` reference the toggle. 

For the last example, the aria-label should be updated to "Options menu" to be consistent with the default text string we will use in the react component.

Additionally, in the last example, the text that is included before the toggle should be updated to "Custom text" to avoid confusion about how that variation is expected to work (i.e. the text doesn't reflect the selected option, but is just some custom text).

**Comment by jgiardino** [comment]
For the icon example, **Options Menu - Plain**, these are the updates we need...
What we have now:
```
<div class="pf-c-options-menu">
  <span id="options-menu-plain-example-label" hidden>Sort methods:</span>
  <button class="pf-c-options-menu__toggle pf-m-plain" id="options-menu-plain-example-toggle" aria-haspopup="listbox" aria-expanded="false" aria-labelledby="options-menu-plain-example-label options-menu-plain-example-toggle" aria-label="Sort by">
    <i class="fas fa-sort-amount-down" aria-hidden="true"></i>
  </button>
  <ul class="pf-c-options-menu__menu" aria-labelledby="options-menu-plain-example-label" hidden>
```
What it should be:
```
<div class="pf-c-options-menu">
  <button class="pf-c-options-menu__toggle pf-m-plain" id="options-menu-plain-example-toggle" aria-haspopup="listbox" aria-expanded="false" aria-label="Sort by">
    <i class="fas fa-sort-amount-down" aria-hidden="true"></i>
  </button>
  <ul class="pf-c-options-menu__menu" aria-labelledby="options-menu-plain-example-toggle" hidden>
```

**Comment by jgiardino** [comment]
For other examples that display text as the toggle label, these are the updates we need...
What we have now:
```
<div class="pf-c-options-menu">
  <span id="options-menu-single-example-label" hidden>Items per page:</span>
  <button class="pf-c-options-menu__toggle" id="options-menu-single-example-toggle" aria-haspopup="listbox" aria-expanded="false" aria-labelledby="options-menu-single-example-label options-menu-single-example-toggle">
    <span class="pf-c-options-menu__toggle-text">
          Options Menu
        </span>
    <i class="fas fa-caret-down pf-c-options-menu__toggle-icon" aria-hidden="true"></i>
  </button>
  <ul class="pf-c-options-menu__menu" aria-labelledby="options-menu-single-example-label" hidden>
```
What it should be:
```
<div class="pf-c-options-menu">
  <button class="pf-c-options-menu__toggle" id="options-menu-single-example-toggle" aria-haspopup="listbox" aria-expanded="false">
    <span class="pf-c-options-menu__toggle-text">
          Options Menu
        </span>
    <i class="fas fa-caret-down pf-c-options-menu__toggle-icon" aria-hidden="true"></i>
  </button>
  <ul class="pf-c-options-menu__menu" aria-labelledby="options-menu-single-example-toggle" hidden>
```
i.e. remove aria-labelledby from `<button>`, and update aria-labelledby in `<ul>`

**Comment by jgiardino** [comment]
@mcarrano The last example shows that Option 2 is the toggle label. Is that intentional or a mistake (e.g. are option menu toggles static or can they change based on user selection)? Or is this example just to show the plain example?
